src/styles/index.scss

```diff
@@ -1,3 +1,73 @@
+$cell-width: 100px;
+$cell-height: 100px;
+$days: (
+  'mon': 0,
+  'tue': 1,
+  'wed': 2,
+  'thu': 3,
+  'fri': 4,
+  'sat': 5,
+  'sun': 6,
+);
```

Comment:
Consider using single quotes for string literals to maintain consistency with the rest of your SCSS code as per the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules).